### ISSUE TITLE
Normative/informative references

### ISSUE THREAD

**Issue by jiceatscion** [scionassociation/scion-cp_I-D#40]
Make sure the references are righttly placed according to [guidelines](https://datatracker.ietf.org/doc/statement-iesg-iesg-statement-normative-and-informative-references-20060419/#:~:text=Normative%20references%20specify%20documents%20that,it%20only%20provides%20additional%20information): 
- Normative references: Needed for the specification; specification is based on it.
- Informative references: Provide background/additional information

Potential changes: 
- [ ] I-D.scion-cppki should be normative 
- [ ] Several RFCs might be informative
- [x] Check if other drafts affected

DP https://github.com/scionassociation/scion-dp_I-D/issues/25 
PKI: https://github.com/scionassociation/scion-cppki_I-D/issues/32

**Comment by nicorusti** [comment]
```
7.1.  Normative References

   [I-D.scion-cp]
              de Kater, C., Rustignoli, N., and S. Hitz, "SCION Control
              Plane", 2024, <https://datatracker.ietf.org/doc/draft-
              dekater-scion-controlplane/>.
              
```

**Comment by nicorusti** [comment]
FIxed with #38
